src/master-playlist-controller.js

```diff
@@ -143,6 +143,11 @@ export class MasterPlaylistController extends videojs.EventTarget {
       timeout: null
     };
 
+    this.on('error', () => {
```

Comment:
We can take this change out if it seems too big, but IMO we should be pausing all loading when we cause a player error.